packages/core/src/builtin-tasks/node.ts

```diff
@@ -23,64 +25,122 @@ const TERMINATION_SIGNALS = ['SIGINT', 'SIGTERM', 'SIGQUIT']
 const BUILTIN_SCRIPTS = ['anyone_can_pay', 'omni_lock', 'simple_udt']
 const DEFAULT_BUILTIN_CONTRACT_DOWNLOAD_BASE_URL =
   'https://github.com/ckb-js/ckb-production-scripts/releases/download/scripts'
+const DEFAULT_CKB_BIN_DOWNLOAD_BASE_URL = 'https://github.com/nervosnetwork/ckb/releases/download'
+const DEFAULT_CKB_BIN_VERSION = 'v0.109.0'
```

Comment:
The latest version can be fetched by

```bash
curl -L \
  -H "Accept: application/vnd.github+json" \
  https://api.github.com/repos/nervosnetwork/ckb/releases/latest
```

Reply:
> The latest version can be fetched by
> 
> ```shell
> curl -L \
>   -H "Accept: application/vnd.github+json" \
>   https://api.github.com/repos/nervosnetwork/ckb/releases/latest
> ```

I see, I will change the way for getting the package information by using this api.